code/modules/admin/admin_verbs.dm

```diff
@@ -657,20 +656,6 @@ var/list/special_pa_observing_verbs = list(
 		src.holder.s_respawn()
 	return
 
-/client/proc/jobbans()
```

Comment:
If you're deleting the interface, how are admins supposed to view current jobbans for a ckey?

Reply:
Player options, for offline players I'm just about to add a replacement for this panel to view them.